server/command.go

```diff
@@ -183,6 +186,95 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 	return &model.CommandResponse{}
 }
 
+func calculateQueItemNumber(args *model.CommandArgs, p *Plugin, hashtag string) (*model.CommandResponse, int) {
+	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &hashtag})
+	if appErr != nil {
+		return responsef("Error calculating list number"), 0
+	}
+	postList := *searchResults.PostList
+	numQueueItems := len(postList.Posts)
+	return nil, numQueueItems + 1
+}
+
+func (p *Plugin) executeCommandReQueue(args *model.CommandArgs) *model.CommandResponse {
+	split := strings.Fields(args.Command)
+
+	if len(split) <= 2 {
+		return responsef("Missing parameters for requeue command")
+	}
+
+	meeting, err := p.GetMeeting(args.ChannelId)
+	if err != nil {
+		return responsef("There was no meeting found for this channel.")
+	}
+
+	oldPostID := split[2]
+	postToBeReQueued, er := p.API.GetPost(oldPostID)
+	if er != nil {
+		return responsef("Error fetching post.")
+	}
```

Comment:
0/5 We should log this error

```suggestion
	postToBeReQueued, appErr := p.API.GetPost(oldPostID)
	if appErr != nil {
		return responsef("Error fetching post.")
	}
```

Reply:
Added Logs for error.